Makefile

```diff
@@ -210,7 +210,7 @@ SEMCONVPKG ?= "semconv/"
 semconv-generate: | $(SEMCONVGEN) $(SEMCONVKIT)
 	[ "$(TAG)" ] || ( echo "TAG unset: missing opentelemetry specification tag"; exit 1 )
 	[ "$(OTEL_SPEC_REPO)" ] || ( echo "OTEL_SPEC_REPO unset: missing path to opentelemetry specification repo"; exit 1 )
-	$(SEMCONVGEN) -i "$(OTEL_SPEC_REPO)/semantic_conventions/." --only=span -p conventionType=trace -f trace.go -t "$(SEMCONVPKG)/template.j2" -s "$(TAG)"
+	$(SEMCONVGEN) -i "$(OTEL_SPEC_REPO)/semantic_conventions/." --only=span,attribute_group,scope -p conventionType=trace -f trace.go -t "$(SEMCONVPKG)/template.j2" -s "$(TAG)"
```

Comment:
Yea, actually probably should be for when metrics and logs are added. I recall having issues in my first attempts with that. I just tried again and only `attribute_group` works. Generating `scope.go` fails:

```
/home/tristan/Devel/opentelemetry-go/.tools/semconvgen -i "../opentelemetry-specification/semantic_conventions/." --only=scope -p conventionType=trace -f scope.go -t ""semconv/"/template.j2" -s "v1.19.0"
panic: unable to render template: exit status 2
usage: gen-semconv [-h] [--debug] [--only ONLY] [--yaml-root folder]
                   [--exclude EXCLUDE]
                   [YAML_FILE ...] {code,markdown} ...
gen-semconv: error: No semantic convention model found!


goroutine 1 [running]:
main.main()
	/home/tristan/go/pkg/mod/go.opentelemetry.io/build-tools/semconvgen@v0.6.0/generator.go:62 +0x4ae
make: *** [Makefile:215: semconv-generate] Error 2
```


I'll have to look tomorrow at what "No semantic convention model" means here.

Reply:
@tsloughter I looked at it and I see "scope" semantic conventions in the specification at this time. I am not sure why you added it, but it looks as it is not needed.

Reply:
I think this:

```suggestion
	$(SEMCONVGEN) -i "$(OTEL_SPEC_REPO)/semantic_conventions/." --only=span -p conventionType=trace -f trace.go -t "$(SEMCONVPKG)/template.j2" -s "$(TAG)"
	$(SEMCONVGEN) -i "$(OTEL_SPEC_REPO)/semantic_conventions/." --only=attribute_group -p conventionType=trace -f attribute_group.go -t "$(SEMCONVPKG)/template.j2" -s "$(TAG)"
```

Plus regenerating the files should be OK.

I wanted to do this on my own but I think that `semconvgen` does not work properly on Windows WSL 🤦 

```
panic: unable to exec /usr/bin/git worktree add /tmp/otel_semconvgen3021531623/input v1.19.0: exit status 128
```

Reply:
I managed to workaround it. @tsloughter are the instructions in `REALEASING.md` working for you? I had to checkout the `otel-specification` repository to a different branch than `v1.19.0` before running `make semconv-generate`. I think it may be a bug in Git for Windows; https://git-scm.com/docs/git-worktree#_bugs 😅

Reply:
Thanks!

Hm, there definitely was a `scope` type, but now searching I only see `type: otel.scope` in the semconvs...

Reply:
nm, `otel.scope` was the prefix not the type. So nothing like `scope` is used as a `type`, not sure why it is part of build tools.